### ISSUE TITLE
Accessibility issue: Share buttons widget lack <button> tags, inoperable with keyboard

### ISSUE THREAD

**Issue by besweeet** [elementor/elementor#11257]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
It appears as though the social share buttons can't be controlled with the keyboard due to the lack of <button> tags, which violated WCAG 2.1.1.

**Comment by besweeet** [comment]
Bump...

**Comment by Bovelett** [comment]
Related to issue #11299 , where there's mention of the div having no tabindex. @joshmarom / @matipojo

**Comment by besweeet** [comment]
I switched to the "AddThis" plugin which doesn't have this issue.
